### DISCUSSION TITLE
Can we add a mount point for textual_inversion?

### DISCUSSION THREAD

**Discussion by davecranwell** [AbdBarho/stable-diffusion-webui-docker#284] [Ideas]
Embeddings can need a degree of trial and error testing to see which stepping gives the best matches. To be able to swap these temporary embeddings with the one in the mounted `/data/embedfings`, then `/textual_inversion` needs adding as a mount too

**Comment by AbdBarho** [comment]
@davecranwell sure, add your mounts here:
https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/ceeac61cb0c328eebe862743c21901dd9d3276a6/docker-compose.yml#L6-L8
and you should be good to go.
